### ISSUE TITLE
[question] Meshroom does not launch when qmlAliceVision is present

### ISSUE THREAD

**Issue by skinkie** [alicevision/Meshroom#985]
**Describe the problem**
I am currently using QT 5.15. I know this is not supported, etc. I am on the meshroom branch **develop**.
I am a problem with launching Meshroom **only** when qmlAliceVision is in the qml searchpath. The application won't launch, and there are no error messages. Removing it, launches the application.

What would help: what part of the application could I force to result into something like a backtrace? I am aware of the  QT_DEBUG_PLUGINS=1 QML_IMPORT_TRACE=1 which surprisingly do not give any hints.

**Comment by fabiencastan** [comment]
There is a problem with Qt 5.15 and the logger that get stuck on the first log.
So it's may be just a log that are blocking it.

**Comment by Julien-Haudegond** [comment]
Hello! Effectively, there is a problem with this line: https://github.com/alicevision/meshroom/blob/develop/meshroom/ui/app.py#L110
```python
qInstallMessageHandler(MessageHandler.handler)
```
If we comment it, it solves the issue. This is not perfect but like this, it works with Qt5.15.

**Comment by skinkie** [comment]
@Julien-Haudegond that is a good find!! I am able to launch!

I am able to crash the application. But I'll open a new ticket for that one :-)

**Comment by skinkie** [comment]
Please keep open, until the fix is merged.

**Comment by skinkie** [comment]
@cbentejac closed or not?

**Comment by cbentejac** [comment]
Yes, the issue with the logger has been solved a while back, so this issue can be fully closed :)
